### PR TITLE
parameterize name

### PR DESCRIPTION
**Pull request by gabemontero**
@bparees Fixes https://github.com/openshift/origin/issues/3571

### CODE REVIEW

**Comment by gabemontero** [comment]
pushed updates based on comments from cakephp-ex

File: openshift/templates/rails-postgresql.json [hunk 1]
```diff
@@ -373,6 +373,12 @@
   ],
   "parameters": [
     {
+      "name": "NAME",
+      "displayName": "Name",
+      "description": "The name assigned to all of the objects defined in this template.",
```
Comment (bparees) [inline openshift/templates/rails-postgresql.json]: frontend

**Comment by gabemontero** [comment]
ok really pushed this time

**Comment by bparees** [comment]
lgtm

**Pull request merged**